composer.json

```diff
@@ -61,7 +61,7 @@
         "predis/predis": "^1.1",
         "symfony/asset": "^3.4|^4.3",
         "symfony/framework-bundle": "^3.4|^4.3",
-        "symfony/lock": "^3.4|^4.3",
+        "symfony/lock": "^3.4|^4.3|^5.3",
```

Comment:
- We are using a shared lock which is from `^5.2` - `"symfony/lock": "^5.2"`. Wont work on older versions.
- Let's drop Symony3 completely from everywhere.
- Bump everywhere else `"symfony/package":"^4.4"`